kie-docker-ui-app/kie-docker-ui-webapp/src/main/java/org/kie/dockerui/backend/util/Timer.java

```diff
@@ -0,0 +1,26 @@
+package org.kie.dockerui.backend.util;
+
+public class Timer {
```

Comment:
Oh yes, i didn't know about it thx for the info, sounds good. It's really not important, as this timer is only used when debug logging is enabled, and I used it just for checking the backend side performance improvements, so i would say that although the fix is really easy, i would add it in my todo list and first do this merge, test that all is ok... and then we can just do this fix later. Anyway, thx, it's a good idea and i have learned something new today :)